extras/crossplane/helmrelease-crossplane.yaml

```diff
@@ -16,14 +16,14 @@ spec:
   install:
     remediation:
       retries: 10
-      remediateLastFailure: False
+      remediateLastFailure: false
   interval: 1m
   targetNamespace: crossplane
   timeout: 10m
   upgrade:
     remediation:
       retries: 10
-      remediateLastFailure: False
+      remediateLastFailure: false
   values:
     giantswarm:
```

Comment:
I think these do not exist anymore: https://github.com/giantswarm/crossplane/blob/main/helm/crossplane/values.yaml as providers are set in the CMC by referencing the bases.

Reply:
This isn't aimed at the providers, this is for crossplane helmrelease install.

- https://fluxcd.io/flux/components/helm/api/#helm.toolkit.fluxcd.io/v2beta1.InstallRemediation
- https://fluxcd.io/flux/components/helm/api/#helm.toolkit.fluxcd.io/v2beta1.UpgradeRemediation

The provider stuff is further down under values and I think that could come out as I don't think it does anything today.

```yaml
    giantswarm:
      kubernetes:
        provider:
          kind: "replaced in kustomization"
      crossplane:
        providers:
          contribAws:
            enabled: false
```